controllers/front/orderSuccess.php

```diff
@@ -301,16 +305,16 @@ private function displayOrderConfirmation($intent)
                 'order-confirmation',
                 true,
                 null,
-                array(
-                    'id_cart' => $id_cart,
-                    'id_module' => (int)$this->module->id,
+                [
+                    'id_cart' => $id_cart ?? 0,
```

Comment:
Not compatible in PHP 5.6
Nevertheless this condition sould be done before using $id_cart